contracts/ITokenListManager.sol

```diff
@@ -11,16 +11,16 @@ interface ITokenListManager {
     }
 
     // Add tokens to tokenlist(whitelist), batch supported
-    function addTokens(address[] calldata tokens) external;
+    function addTokens(address[] calldata tokens, TokenType tokenType) external;
```

Comment:
Why not detect token type via ERC165? For NFT, token should support IERC721 or IERC1155. Otherwise, it is ERC20 that do not support IERC165.

Reply:
Solved

Reply:
No code change found for this comment.

Reply:
Detect token type via ERC165 right now， and ERC20 as well.